packages/create-block/templates/javascript/edit.jsx.mustache

```diff
@@ -0,0 +1,20 @@
+import { __ } from '@wordpress/i18n';
+import { useBlockProps } from '@wordpress/block-editor';
+
+import './index.scss';
+
+/**
```

Comment:
Wondering if we need this comment also - I'd be in favor of removing it in favor of docs that are part of this package about how it works, rather than having this as part of every edit file

Reply:
Yeah, I get that. I am hesitant to put in documentation about how the edit function works but instead rely on the developer to reference the block editor handbook when authoring blocks.

Reply:
Some ESlint and Typescript configurations require functions to have docblocks. That said this could be consolidated.

Reply:
I made the comments much more contextual referencing the block with the namespace/block-name convention. That way the dev will be able to know what block the code applies to, rather than generalized info about the file in block development.